screens/LoadingScreen.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, StyleSheet, ActivityIndicator } from 'react-native';
+import firebase from 'firebase';
+
+class LoadingScreen extends React.Component {
+    componentDidMount() {
```

Comment:
If you'd like, you can use react-hooks instead of plain react. It basically allows you to use state in functional components. It's apparently faster and you get to say I know this tech in your resume.

Reply:
done